components/service/glean/metrics.yaml

```diff
@@ -147,3 +147,55 @@ glean.internal.metrics:
     notification_emails:
     - telemetry-client-dev@mozilla.com
 
+glean.error:
+  out_of_range:
+    type: counter
+    labeled: true
+    description:
+      Counts the number of times a metric was set to an out-of-range value.
+      The labels are the `category.name` identifier of the metric.
+    bugs:
+      - 1499761
+    data_reviews:
+      - TODO
```

Comment:
Thanks for filing data review! Can you update these `data_reviews` fields to point to the bug you filed?

Reply:
Sure thing.  Done.